src/main/scala/org/allenai/deep_qa/experiments/SciQDatasetExperiments.scala

```diff
@@ -0,0 +1,302 @@
+    )
+  }
+
+  def combinedSciQOmnibusGradeEightExperiment(
+    name: String,
+    modelParams: JValue,
+    trainingDataset: JValue=combinedSciQTrainOmnibusEightTrainDataset
+  ): JValue = {
+    experiment(
+      name,
+      modelParams,
+      trainingDataset,
+      omnibusEightDevReadingComprehensionDataset
+    )
+  }
+
+  val attentionSumReader: JValue = {
+    Models.attentionSumReader merge
+    (("patience" -> 1) ~
+      ("preferred_backend" -> "theano") ~
+      ("encoder" ->
+        ("default" ->
+          ("type" -> "bi_gru")~
+          ("output_dim" -> 384)
+        )
+      ) ~
+      ("seq2seq_encoder" ->
+        ("default" ->
+          ("type" -> "bi_gru") ~
+          ("encoder_params" ->
+            ("output_dim" -> 384)
+          ) ~
+          ("wrapper_params" -> JObject())
+        )
+      ) ~
+      ("optimizer" ->
+        ("type" -> "adam") ~
+        ("clipnorm" -> 10.0) ~
+        ("lr" -> 0.0005)
+      ) ~
+      ("embedding_dropout" -> 0.0) ~
+      ("patience" -> 0) ~
+      ("embedding_size" -> 256) ~
+      ("num_epochs" -> 5)
+    )
+  }
+
+  val gatedAttentionReader: JValue = {
+    Models.gatedAttentionReader merge
+    (("embedding_size" -> 100) ~
+      ("pretrained_embeddings_file" -> "/efs/data/dlfa/glove/glove.6B.100d.txt.gz") ~
+      ("fine_tune_embeddings" -> false) ~
+      ("project_embeddings" -> false) ~
+      ("num_gated_attention_layers" -> 3) ~
+      ("patience" -> 0) ~
+      ("preferred_backend" -> "theano") ~
+      ("encoder" ->
+        ("question_final" ->
+          ("type" -> "bi_gru")~
+          ("output_dim" -> 128)
+        )
+      ) ~
+      ("seq2seq_encoder" ->
+        ("question_0" ->
+          ("type" -> "bi_gru") ~
+          ("encoder_params" ->
+            ("output_dim" -> 128)
+          ) ~
+          ("wrapper_params" -> JObject())
+        ) ~
+        ("document_0" ->
+          ("type" -> "bi_gru") ~
+          ("encoder_params" ->
+            ("output_dim" -> 128)
+          ) ~
+          ("wrapper_params" -> JObject())
+        ) ~
+        ("question_1" ->
+          ("type" -> "bi_gru") ~
+          ("encoder_params" ->
+            ("output_dim" -> 128)
+          ) ~
+          ("wrapper_params" -> JObject())
+        ) ~
+        ("document_1" ->
+          ("type" -> "bi_gru") ~
+          ("encoder_params" ->
+            ("output_dim" -> 128)
+          ) ~
+          ("wrapper_params" -> JObject())
+        ) ~
+        ("document_final" ->
+          ("type" -> "bi_gru") ~
+          ("encoder_params" ->
+            ("output_dim" -> 128)
+          ) ~
+          ("wrapper_params" -> JObject())
+        )
+      ) ~
+      ("optimizer" ->
+        ("type" -> "adam") ~
+        ("clipnorm" -> 10.0) ~
+        ("lr" -> 0.0005)
+      ) ~
+      ("embedding_dropout" -> 0.0) ~
+      ("patience" -> 0) ~
+      ("num_epochs" -> 5)
+    )
+  }
+
+
+  def main(args: Array[String]) {
+    runASReaderOmnibusEightExperiment
```

Comment:
I think the convention is that if calling a method has side effects, you should include the parens, even if they're not strictly necessary.

http://docs.scala-lang.org/style/method-invocation.html
http://docs.scala-lang.org/style/naming-conventions